code/game/machinery/launch_pad.dm

```diff
@@ -192,35 +192,24 @@
 		return ..()
 
 //Briefcase item that contains the launchpad.
-/obj/item/briefcase_launchpad
-	name = "briefcase"
-	desc = "It's made of AUTHENTIC faux-leather and has a price-tag still attached. Its owner must be a real professional."
-	icon = 'icons/obj/storage.dmi'
-	icon_state = "briefcase"
-	lefthand_file = 'icons/mob/inhands/equipment/briefcase_lefthand.dmi'
-	righthand_file = 'icons/mob/inhands/equipment/briefcase_righthand.dmi'
-	flags_1 = CONDUCT_1
-	force = 8
-	hitsound = "swing_hit"
-	throw_speed = 2
-	throw_range = 4
-	w_class = WEIGHT_CLASS_BULKY
-	attack_verb = list("bashed", "battered", "bludgeoned", "thrashed", "whacked")
-	resistance_flags = FLAMMABLE
-	max_integrity = 150
+/obj/item/storage/briefcase/launchpad
 	var/obj/machinery/launchpad/briefcase/pad
 
-/obj/item/briefcase_launchpad/Initialize()
+/obj/item/storage/briefcase/launchpad/Initialize()
 	. = ..()
 	pad = new(src)
```

Comment:
@spacemaniac is there a difference between new(null) and new()?

Reply:
Not in practice, the former might be more readable

Reply:
Do you know how I can close the briefcase's storage after it's been deployed with attack_self()? I've tried calling close() on it but wasn't successful.

Reply:
Give a try to `SEND_SIGNAL(briefcase, COMSIG_TRY_STORAGE_HIDE_ALL)` where `briefcase` might be `src` depending on where you write the code

Reply:
That worked well, thanks for the help!
I've ran into two issues: 
1) The briefcase's Initialize() location check QDELs the launchpad since it now spawns in nullspace instead of the briefcase. Just changing the checked location to "null" doesn't seem to work.
2) The launchpad doesn't register the briefcase item it's spawned with; instead it's `var/briefcase = null`